README.md

```diff
@@ -118,3 +157,9 @@ For releasing a new version of the extension, follow the steps outlined in
 - Download from:
   - [Chrome Store](https://chrome.google.com/webstore/detail/substrate-connect-extensi/khccbhhbocaaklceanjginbdheafklai)
   - [Mozilla Addons](https://addons.mozilla.org/en-US/firefox/addon/substrate-connect/)
+
+/*
+This makes me wonder what we should do with https://substrate.io/developers/substrate-connect/.
+
+My assumption was that we'd move away from this extension (deprecating it eventually), and we'll move towards a world in which wallets can expose light clients, and then lilbraries like PAPI will connect or instantiate one as necessary. (I do wonder what the case is when multiple wallets expose a light client though; which would PAPI pick? We couldn't necessarily trust them after all)
```

Comment:
its up to the user to choose. similar to when you connect to dapps in ethereum a popup shows up and u can choose metamask, rabby, coinbase wallet, etc...

Reply:
Yeah good point; that makes sense! User chooses the extension they trust and then it's all gravy :)